src/messaging/ReliableMessageMgr.cpp

```diff
@@ -133,6 +133,13 @@ void ReliableMessageMgr::ExecuteActions()
         if (!rc || entry.nextRetransTimeTick != 0)
             continue;
 
+        if (entry.retainedBuf.IsNull())
+        {
+            // Discard the current entry if the retained buffer is null
```

Comment:
The comment seems to re-read the code without telling me 'why' this is done. 
Could you rephrase to make the comment explain the code rather than re-reading it? Could also delete it, howver in this case I believe it may be valuable for the reader of the code to understand why entry may have an empty retained buffer and why clearing is approrpiate.

Could be `empty retaining buffer means CRMP has nothing to resend and entry is invalid` ... but is this correct? why would we have an entry with nextRetransTick set if the retained buffer is null?

Reply:
I'll update the comment.